packages/unified/src/utilsv5.ts

```diff
@@ -46,7 +48,10 @@ import { hashtags } from "./remark/hashtag";
 import { noteRefsV2 } from "./remark/noteRefsV2";
 import { userTags } from "./remark/userTags";
 import { wikiLinks, WikiLinksOpts } from "./remark/wikiLinks";
-import { DendronASTDest } from "./types";
+import { DendronASTDest, UnistNode } from "./types";
+import path from "path";
+import fs from "fs-extra";
```

Comment:
We can't introduce fs-extra in unified or else the web extension will stop working.  We'll need to move this utility to a different package.  This might be a bit tricky to do though because it looks like this gets referenced elsewhere in the unified parsing code.

Reply:
This will be phase two